clang/test/Driver/sycl-sanitize-target.cpp

```diff
@@ -0,0 +1,25 @@
+// UNSUPPORTED: system-windows
+
+// RUN: %clangxx -fsycl -fsanitize=address -fsanitize-target=both %s -### 2>&1 \
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsanitize=address %s -### 2>&1 \
```
Default is when it is not specified, right?